Tests/SwiftSyntaxMacroExpansionTest/PeerMacroTests.swift

```diff
@@ -151,7 +151,7 @@ final class PeerMacroTests: XCTestCase {
           Task {
             completionHandler(await f(a: a, for: b, value))
           }
-        }
+          }
```

Comment:
Okay, I removed trailing and leading newline at it got fixed.. Dunno why tough? 
Should I dig more into it.

Reply:
Where did you remove the trailing and leading newline?

Reply:
Inside: Tests/SwiftSyntaxMacroExpansionTest/PeerMacroTests.swift

Reply:
I think it would be nice to understand why the behavior changed.

Reply:
It happens in the new `transferTrailingTrivaFromEndOfFileIfPresent`
The raw input is `transferTrailingTrivaFromEndOfFileIfPresent` and we return `"\n  Task {\n    completionHandler(await f(a: a, for: b, value))\n  }\n"`

So I think it's because we transfer the leading trivia to the trailing part.
And because it have a newline at the end, we have double newline.

Reply:
You nerd-sniped me into this one: I’m fixing the issue in https://github.com/apple/swift-syntax/pull/2428

Reply:
The changes you made in this PR still make sense to me. So good to go 👍🏽

Reply:
> You nerd-sniped me into this one: I’m fixing the issue in https://github.com/apple/swift-syntax/pull/2428

Sorry 🤣